src/main/java/frc/trigon/robot/subsystems/arm/Arm.java

```diff
@@ -0,0 +1,4 @@
+package frc.trigon.robot.subsystems.arm;
+
+public class Arm {
```

Comment:
Now, start working on the actual subsystem. 
Start by recreating this as a singleton, and try creating as much of the logic as you can.

Reply:
what's a singleton?

Reply:
When you create a new subsystem using the FRC plugin you have an option for singleton, like you did in the collector...